pgdog/src/backend/pool/connection/mirror/mod.rs

```diff
@@ -71,6 +72,19 @@ impl Mirror {
     /// Handler for sending queries to the background task.
     ///
     pub fn spawn(cluster: &Cluster) -> Result<MirrorHandler, Error> {
+        let config = config();
+        Self::spawn_with_config(
+            cluster,
+            config.config.general.mirror_exposure,
```

Comment:
It looks like we're using the general mirror settings. Are we then ignoring the ones set in the `[[mirrors]]` config?

Reply:
I think we can keep the general settings and let the ones set on individual mirrors be override the general ones. Maybe make them optional on the `[[mirrors]]` config and only use them if they are set? Seems like a good backwards-compatible change.